counterparty-core/counterpartycore/server.py

```diff
@@ -576,6 +579,14 @@ def start_all(catch_up="normal"):
 
         db = initialise_db()
 
+        telemetry_daemon = TelemetryDaemon(
+            interval=60,
+            collector=TelemetryCollectorLive(db=db),
+            client=TelemetryClientLocal(),
+        )
+
+        telemetry_daemon.start()
+
```

Comment:
shouldn't we call `.stop()` in `finally:` clause ?

Reply:
Good catch.  Yes!

Reply:
Actually, yeah, I think it's cleaner just to not handle it explicitly.  Otherwise i need to do something like this in `stop`:

```
    def stop(self):
        self.is_running = False
        try :
            self.thread.join()
        except KeyboardInterrupt:
            pass
```